CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ### Changes since v0.13
 
 - Change `Lucky::BaseApp` to `Lucky::BaseAppServer`
+- Many text helpers now return a `String` instead of appending to the view (`cycle`, `excerpt`, `highlight`, `pluralize`, `time_ago_in_words`, `to_sentence`, `word_wrap`) [#781](https://github.com/luckyframework/lucky/pull/781)
```

Comment:
Might be worth putting in bold `**Breaking change** - Many text...` since this will definitely break stuff using it